### PR TITLE
Resolves #2986 reCAPTCHA v.1 drop

### LINKED ISSUE #2986: ReCaptcha v1 support being dropped
Google is dropping support for Recaptcha v1 so we should likely remove this for .16

### PR DESCRIPTION
**Pull request by effone**
Attempt to comply: https://github.com/mybb/mybb/issues/2986

To Do:
- [x] Remove reCAPTCHA settings
- [x] Remove reCAPTCHA templates
- [x] Remove all template references in scripts
- [x] Remove all checks and processes related to reCAPTCHA generation / refresh
- [x] Remove abandoned language variables from `global.lang.php`
- [x] Alter `optionscode` for `captchaimage` in existing `settings` table to remove `reCAPTCHA` _(upgrade)_
- [x] Check `value` of `captchaimage` in existing `settings` table and if it is set to **2** _(upgrade)_
                    - reset value to **1**
                    - remove values of `captchaprivatekey` & `captchapublickey`

### CODE REVIEW

File: inc/class_captcha.php [hunk 1]
```diff
@@ -104,11 +104,7 @@ function __construct($build = false, $template = "")
 		{
 			$this->captcha_template = $template;
 
-			if($this->type == 2)
-			{
-				$this->captcha_template .= "_recaptcha";
-			}
-			elseif($this->type == 4){
+			if($this->type == 4){
```
Comment (euantorano) [inline inc/class_captcha.php]: Code style - can you move the `{` to a new line?

File: install/resources/upgrade44.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
```
Comment (euantorano) [inline install/resources/upgrade44.php]: We already have an upgrade script for 1.8.16: https://github.com/mybb/mybb/blob/feature/install/resources/upgrade43.php - the naming is super confusing, I know 😞

Comment (effone) [reply install/resources/upgrade44.php]: Was not familiar with upgrade script system. Sorry. Will merge to that.

File: inc/class_captcha.php (hunk 1 above)
Comment (effone) [reply inc/class_captcha.php]: Done.

File: install/resources/upgrade44.php (hunk 2 above)
Comment (effone) [reply install/resources/upgrade44.php]: @euantorano 44 unlinked. code transferred to 43

**Comment by effone** [comment]
@euantorano does it worth digging as Google clearly stated to "Please **register a new key** and upgrade to **v2.**". Old keys will not work thats because I have removed those from db.

Source: https://developers.google.com/recaptcha/docs/faq

**Comment by euantorano** [comment]
Alright, sounds good to me. Good work!

**Comment by effone** [comment]
@euantorano I have cleared the conflict, however `post_captcha_recaptcha,` has to be removed from `newreply.php` and `newthread.php` top `$templatelist`.

**Comment by Sama34** [comment]
Can't you get them removed here @effone ?

**Comment by effone** [comment]
@Sama34 I tried (last two commits) but it has to be done after merge. If I do that now conflict will arrive again.

**Pull request merged**